scimono-server/src/main/java/com/sap/scimono/entity/Meta.java

```diff
@@ -52,6 +55,7 @@
   private final String version;
   private final Set<String> attributes;
   private final String resourceType;
+  private final Map<String, Object> additionalAttributes;
```

Comment:
What is the reason to store the names and **the attributes** in the Meta? Isn't it enough to store just the names of the attributes?

Reply:
These are the additional returned values. They are allowed only to be set by the service provider. 
Example from the draft:
```json
{
      "id": "c3a26dd3-27a0-4dec-a2ac-ce211e105f97",
      "schemas": ["urn:ietf:params:scim:schemas:core:2.0:Group"],
      "displayName": "Group A",
      "meta": {
        "resourceType": "Group",
        "created": "2011-08-01T18:29:49.793Z",
        "lastModified": "2011-08-01T18:29:51.135Z",
        "location":
"https://example.com/v2/Groups/c3a26dd3-27a0-4dec-a2ac-ce211e105f97",
        "version": "W\/\"mvwNGaxB5SDq074p\"",
        "members.cnt":1
      },
```